autospec/build.py

```diff
@@ -143,7 +143,7 @@ def failed_pattern(line, pattern, verbose, buildtool=None):
                 must_restart += buildreq.add_buildreq(config.gems[s], cache=True)
             else:
                 print("Unknown ruby gem match", s)
-        elif buildtool == 'maven':
+        elif buildtool == 'maven' or buildtool == 'gradle':
```

Comment:
So if the buildtool is gradle it will also add mvn_provide, is that expected?

Reply:
Yes, that is expected, because we are installing all artifacts to the same repository tree, and every jar, pom, or signature artifact will have a `mvn()` virtual provide added for it.

Reply:
On second thought, this change is a no-op, because `maven` is still specified for the new gradle fail patterns.

Reply:
Yup, it would use all mvn pkgs just like any other maven packages.